rancher2/schema_auth_config_keycloak.go

```diff
@@ -48,6 +48,10 @@ func authConfigKeyCloakFields() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Required: true,
 		},
+		"entity_id": {
+			Type:     schema.TypeString,
+			Optional: true,
```

Comment:
If `entity_id` has a default value, one of 2 options should be implemented to avoid false diff:
* Add default value on the argument definition. `Default: "<DEFAULT_VALUE>",`
* Define the argument also as computed. `Computed: true,`

Reply:
If not provided, the Rancher API will default to what is [documented](https://rancher.com/docs/rancher/v2.x/en/admin-settings/authentication/keycloak/), that implies to the user it is `Computed`, but it `Computed` by `Rancher`, not `Terraform` (this is probably where I am a bit confused).

Will the following be correct?

```	
Type:     schema.TypeString,			
Default: "https://yourRancherHostURL/v1-saml/keycloak/saml/metadata"
Optional: true,
Computed: true,
```

Reply:
(answering my own question) I guess not, the "Default" is generated by `Rancher` with it's known `yourRancherHostURL`, not sure now what to provide as default in Terraform case.

Reply:
Indeed, a `computed` tf argument use to be computed externally, in that case at Rancher side. If the field is not computed, the default value should be specified at definition to avoid false diff on resource comparisson